.github/workflows/integration-test-config-older-k8s/action.yaml

```diff
@@ -0,0 +1,38 @@
+name: Cluster Setup
```

Comment:
This looks like a bunch of extra CI to support 1.17. To what version of K8s will we need this workaround? Did you say we would have to bump to K8s 1.20+, or which version?

Reply:
not exactly an extra CI run, the count is the same just they will refer to different files for configuration. But I think it will be better If we could upgrade our older k8s version(I don't have an exact version, depends on how ci behaves) for CI stability and less complexity to maintain older versions

Reply:
Yeah I was just referring to the extra complexity of CI implementation, at some point we have to raise our min version to simplify it

Reply:
exactly, so what version you are suggesting moving to? IMO anything around 1.20.0 should be good to manage

Reply:
Good to see the CI is green now, this will be good to still have the older support and backport to 1.9. For the min version change, let's discuss on #10809.